docs/HACKING.md

```diff
@@ -29,6 +29,9 @@ If you already have a configured cluster, move on to [Build](#build)
 
 ## Build
 
+### Prerequisites
+- [install the operator-sdk on your machine](https://sdk.operatorframework.io/docs/installation/). WMCO uses version 1.15.0 
```

Comment:
You can just say `Install Operator SDK 1.15.0 on your machine` and link to https://v1-15-x.sdk.operatorframework.io/docs/installation/